config/config.go

```diff
@@ -48,6 +49,29 @@ func (s Secret) MarshalJSON() ([]byte, error) {
 	return json.Marshal(secretToken)
 }
 
+type Header map[string][]Secret
```

Comment:
this is added mostly to be able to test the conversion to http.Header and the marshaling / unmarshaling.